Detectors/EMCAL/base/include/EMCALBase/Digit.h

```diff
@@ -51,19 +52,23 @@ class Digit : public DigitBase
     return lhs;
   }
 
-  Int_t GetTower() const { return mTower; }
-  void SetTower(Int_t tower) { mTower = tower; }
+  Short_t GetTower() const { return mTower; }
+  void SetTower(Short_t tower) { mTower = tower; }
 
   Double_t GetAmplitude() const { return mAmplitude; }
   void SetAmplitude(Double_t amplitude) { mAmplitude = amplitude; }
+    
+  Int_t GetLabel() const { return mLabel; }
+  void SetLabel(Int_t label) { mLabel = label; }
 
   void PrintStream(std::ostream& stream) const;
 
  private:
   friend class boost::serialization::access;
 
-  Int_t mTower;        ///< Tower index (absolute cell ID)
+  Short_t mTower;      ///< Tower index (absolute cell ID)
```

Comment:
please arrange data member in decreasing size/alignment order. This should make the Digit smaller.

Reply:
OK, I can fix it.